codecept.conf.js

```diff
@@ -66,6 +66,7 @@ exports.config = {
     addStatementOfServiceEventPage: './e2e/pages/events/addStatementOfServiceEvent.page.js',
     uploadC2DocumentsEventPage: './e2e/pages/events/uploadC2DocumentsEvent.page.js',
     draftStandardDirectionsEventPage: './e2e/pages/events/draftStandardDirectionsEvent.page.js',
+    uploadC21OrderEventPage: './e2e/pages/events/uploadC21OrderEvent.page.js'
```

Comment:
is it 'upload' or 'create'. You've used both.

Reply:
Create - fixed